internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -198,8 +198,14 @@ func (c *scorecardCmd) run() (err error) {
 	if c.list {
 		scorecardTests = o.List()
 	} else {
+		runnerSA := ""
+		if o.Config.ServiceAccount != "" {
+			runnerSA = o.Config.ServiceAccount
+		} else {
+			runnerSA = c.serviceAccount
```

Comment:
Would c.serviceAccount always hold a value? Could it be empty?

Reply:
Yes, if it is not set by the user in the CLI, it defaults to "default" [here](https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/scorecard/cmd.go#L81)

Reply:
Go it, thanks for clarifying.